go.mod

```diff
@@ -196,3 +196,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace github.com/longhorn/longhorn-instance-manager v1.4.0-rc1.0.20230613094429-85c7d1d7f70b => github.com/derekbit/longhorn-instance-manager v0.0.0-20230614171210-9aa89d527ac6
```

Comment:
Should remove this replace before merging?

Reply:
Yes, waiting for the merge of https://github.com/longhorn/longhorn-spdk-engine/pull/53 and https://github.com/longhorn/longhorn-instance-manager/pull/248